src/cast.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+import {Tensor} from '../src/lib/tensor.js';
+
+/**
+ * Cast each element in the input tensor to the target data type.
+ * @param {Tensor} input
+ * @return {Tensor}
+ */
+
+export function cast(input, type) {
+  let outputArray;
+  switch (type) {
+    case 'float32':
+      outputArray = new Float32Array(input.data);
+      break;
+    case 'int32':
+      outputArray = new Int32Array(input.data);
+      break;
+    case 'uint32':
+      outputArray = new Uint32Array(input.data);
+      break;
+    case 'int64':
```

Comment:
Please fix this error "TypeError: Cannot convert a BigInt value to a number" of CI.